airbyte-integrations/connectors/source-intercom/unit_tests/test_control_rate_limit.py

```diff
@@ -0,0 +1,44 @@
+#
```

Comment:
this file was reset by reverting [this PR](https://github.com/airbytehq/airbyte/pull/23013)